examples/e2e/experimental/src/app/use-cache/ssr/page.tsx

```diff
@@ -9,12 +9,15 @@ export default async function Page() {
 		<div>
 			<h1>Cache</h1>
 			<p>{_headers.get("accept") ?? "No accept headers"}</p>
+			cached:
 			<Suspense fallback={<p>Loading...</p>}>
 				<FullyCachedComponent />
 			</Suspense>
+			cached (with tag):
 			<Suspense fallback={<p>Loading...</p>}>
 				<FullyCachedComponentWithTag />
 			</Suspense>
+			isr:
```

Comment:
```suggestion
			<h2>isr:</h2>
```

I'm not sure if it's helpful, but it might make the page easier to manually check? 🤷